pom.xml

```diff
@@ -43,8 +43,8 @@
     </developers>
     <distributionManagement>
         <repository>
-            <id>bintray</id>
-            <url>https://api.bintray.com/maven/yanaga/maven/sqldroid</url>
+            <id>ngti.sonatype</id>
+            <url>https://repo.ngti.nl/nexus/content/repositories/snapshots/</url>
```

Comment:
Are you going to upload only snapshots ? I think we should make a release to prevent changes in the library.

Reply:
Changed to release.